src/node/Params.hs

```diff
@@ -94,9 +96,11 @@ getNodeParams args@Args {..} systemStart = do
                 (CLI.bitcoinDistr commonArgs)
                 (CLI.richPoorDistr commonArgs)
                 (CLI.expDistr commonArgs)
-    let npGenesisUtxo
-            | isDevelopment = genesisUtxo Nothing (devAddrDistr devStakeDistr)
-            | otherwise = genesisUtxoProduction
+    let npGenesisTxpCtx = mkGenesisTxpContext $
+            if isDevelopment
+            then genesisUtxo Nothing (devAddrDistr devStakeDistr)
+            else genesisUtxo (Just genesisProdBootStakeholders)
+                                genesisProdAddrDistribution
```

Comment:
Why did you change `genesisUtxoProduction` to `genesisUtxo (Just genesisProdBootStakeholders) genesisProdAddrDistribution`?